java/client/src/main/java/glide/utils/ArrayTransformUtils.java

```diff
@@ -128,7 +129,10 @@ public static <T> Map<String, T[][]> castMapOf2DArray(
             return null;
         }
         return mapOfArrays.entrySet().stream()
-                .collect(Collectors.toMap(Map.Entry::getKey, e -> castArrayofArrays(e.getValue(), clazz)));
+                .collect(
+                        HashMap::new,
```

Comment:
why this change? to accept nulls?

Reply:
yes.  XREADGROUP returns maps will null values (gross!)

Reply:
Do you have a test for that case?

Reply:
I do